awesome_owl/static/src/TodoList/todolist.js

```diff
@@ -0,0 +1,57 @@
+import { Component, onMounted, useRef, useState } from '@odoo/owl'
+import { TodoItem } from '../TodoItem/todoitem';
+
+
+export class TodoList extends Component {
+    static template = "awesome_owl/TodoList"
+    static components = { TodoItem }
+
+    setup() {
+        this.state = useState({ 
+            todos: [
+                { id: 0, description: "buy Milk", isCompleted:true },                
+                { id: 1, description: "Learn Owl", isCompleted: false },
+                { id: 2, description: "Build a project", isCompleted: true },
+            ],
+        });
+
+        this.inputRef = useRef('todoInput')
+
+        onMounted(() => {
+            this.inputRef.el.focus();
+        })
+    }
+
+    addTodo(event) {
+        if(event.keyCode == 13) {
+            const tododata = event.target.value.trim();
```

Comment:
```suggestion
            const todoData = event.target.value.trim();
```
Please follow this [guide for js](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript#variable_names)